autoextract_poet/mixins.py

```diff
@@ -0,0 +1,19 @@
+from web_poet.mixins import ResponseShortcutsMixin
+from web_poet.page_inputs import ResponseData
+
+
+class AutoExtractResponseShortcutsMixin(ResponseShortcutsMixin):
+
+    @property
+    def response(self):
+        # FIXME: raise a custom Exception or create a separate method for errors
+        data = self.autoextract_response.data
+
+        # FIXME: pageType could be different to item key in the response
+        pageType = data["query"]["userQuery"]["pageType"]
+
+        # FIXME: maybe better to have a AEResponseData to not confuse with the ResponseData semantic
```

Comment:
Ideally, we should have an `AEResponseData` that could be injectable. But at the same time we should group requests to AutoExtract. See my comment above.

Reply:
The mixin was entirely removed for now.